README.md

```diff
@@ -1,24 +1,69 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] I was unable to create greetings because you are missing the greeting model in your `app/models` file. To check other features I had to create the model myself. You need to create the file as well.

  ![greetings](https://user-images.githubusercontent.com/36542986/184873326-ae86d5ab-e97e-4062-8c2a-9588820d5b6f.jpg)